app/src/main/res/values/strings.xml

```diff
@@ -138,7 +138,7 @@
     <string name="activity_description_date">date</string>
     <string name="activity_description_duration">Duration</string>
     <string name="activity_description_address">Address</string>
-    <string name="activity_description_description">Address</string>
```

Comment:
Thanks for fixing this, it didn't make much sense when I saw it in the app

Reply:
Yes !